include/alpaka/onAcc/scope.hpp

```diff
@@ -6,8 +6,29 @@
 
 #include <string>
 
+/**
+ * @brief Provides scopes for atomic and memory fence operations, analogous to NVIDIA CUDA's atomic and fence scopes.
+ *
+ * This namespace defines the visibility scopes for atomic operations and memory fences,
+ * which control the visibility of memory operations across threads, blocks, and devices.
+ * The provided scopes are:
+ * - Block: Visibility within a thread block.
+ * - Device: Visibility across all thread blocks on the same device.
+ * - System: System-wide visibility, mapped to the strongest available atomic/fence by the backend.
+ *
+ * @see alpaka::atomic::atomicAdd, alpaka::memory::threadFence
```

Comment:
```suggestion
 * @see alpaka::onAcc::atomicAdd, alpaka::onAcc::memFence
```
- `atomicAdd` is not located in the `alpaka::atomic` namespace
- There is no `threadFence` function/struct. Do you mean `alpaka::onAcc::memFence`

Reply:
fixed, can be merged after the CI passed